{
	"id": "CVE-2021-40839",
	"vulnerabilities": [
		{
			"id": "CVE-2021-40839",
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2021-40839"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.18",
			"vulnerable": true,
			"package": {
				"name": "py3-rencode",
				"repositories": [
					"community"
				],
				"architectures": [
					"aarch64",
					"armhf",
					"armv7",
					"ppc64le",
					"s390x",
					"x86",
					"x86_64"
				]
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "1.0.6-r7"
					}
				],
				"fixed": [
					"1.0.6-r7"
				]
			}
		}
	],
	"data_source": "alpine-secdb"
}
